src/test/java/org/junitpioneer/jupiter/StdIoExtensionTests.java

```diff
@@ -122,40 +134,45 @@ class ResettingTests {
 		@Test
 		@ReadsStdIo
 		@Order(1)
-		@DisplayName("1: System.in and System.out is untouched")
+		@DisplayName("1: System.in, System.err and System.out is untouched")
```

Comment:
Nitpick: if you default to the order `in`, `err`, `out`, I would also use that order in the code, e.g., the assertions in `untouched` or the parameters in `redirected`.

Reply:
Makes sense. 👍